2019-12-11-all-hands-meeting.md

```diff
@@ -0,0 +1,149 @@
+# 2019-12-11 Hypha Worker Co-op: All Hands Meeting
+
+<sup>[from template][template] | [:notebook: Meetings][meetings] | [:date: Calendar][calendar] | [:white_check_mark: Tasks][tasks] | [:cat: Code][gh] | [:open_file_folder: Drive][gdrive]</sup>
+
+Time:       Wed, Dec 11, 2019 @ 5-6pm  
+Location:   https://jitsi.hypha.coop/hyphacoop  
+Attending:  bl, uv, pc, dc, el, yj  
+Cultivator: bl  
+Notetaker:  el :raising_hand: Next up: el, dc, bl, yj, uv, pc, gi
+
+## Agenda
+
+| Item                                        | Time (min) |
+|:--------------------------------------------|-----------:|
+| [Personal Checkins](#Personal-Checkins)     |          5 |
+| [Task Board Review](#Task-Board-Review)     |         25 |
+| --- [Working Group Updates](#Working-Group-Updates) |    |
+| --- [Project Updates](#Project-Updates)     |            |
+| [Announcements](#Announcements)             |          5 |
+| [Discussions](#Discussions)                 |         20 |
+| [Process Checkout](#Process-Checkout)       |          5 |
+
+## Notes
+
+### Personal Checkins
+
+- One thing you are dying to wrap up before the year end
+- yj
+    - One thing have to wrap up have to upgrade server and windows 7 to 10
+    - Microsoft no longer support win7/server 2k8r2
+- bl
+    - Wrap up all the conference things dweb, our networks all the devlivables
+- pc
+    - Payroll! Excited to have that chugging along
+- uv
+    - Project since auguest that being wrapped up tomorrow
+- el
+    - Hypha open infra stuff
+- dc
+    - I'm excited to wrap up marking! Some other life things this week
+
+### Task Board Review
+
+- Archive _Done_ tasks :tada:
+- Review [HIGH `[priority-★★★]`][l-pri-hi]
+    - #15 need to discuse with the group
+    - #119 patcon: still needs pipeline that matches our actual thing. +1 ben for tomorrow looking
+    - #154
+    - #156 update below
+    - #134 ongoing
+    - #143 dc need to return the contract
+    - #131 set up WG call for planning
+    - #86 Done
+    - #127 el: review and close
+    - #145 pc thinks its done now
+- Triage new tasks
+- missing labels [`wg:NULL`][l-none] | [`[priority-NULL]`][l-pri-none]
+
+### Working Group Updates
+
+- [Business Planning][biz-wg] :white_check_mark: [`wg:business-planning`][l-biz]
+    - Meeting tomorrow for 2 hours in call
+    - pc and bl put some leads into CRM, have a look before meeting if you get a chance
+- [Finance][fin-wg] :white_check_mark: [`wg:finance`][l-fin]
+    - patcon: #156 Payroll. Finance WG decided to go with Wagepoint. Registered. CRA payroll program account is now set up. Currently doing Wagepoint setup.
+    - Decided not to go with QB payroll and using wagepoint
+    - decide on vacation time 
+    - ex. $100 from client, employee gets $75 + (4% of $75)
+        - vacation accrual. pay out weekly for now, but is short-term. expect to start withholding and manage in medium future, e.g. before year-end.
```

Comment:
```suggestion
        - vacation accrual will be paid out weekly for now, but this is short-term. expect to allow option for withholding and manage in medium future, e.g. before year-end.
```

Reply:
Just to clarify -- pretty sure we have to offer both options and that is what I understood @YurkoWasHere to have said